src/hooks/useTableSort.js

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState } from 'react';
+
+function useTableSort({ sortFields, defaultSortOption }) {
+  const [sortOption, setSortOption] = useState();
+
+  // get the sort option values from the URL, if available
```

Comment:
Should we remove the comments regarding storing sort in the URL? Or is that a future plan to include for this app?

Reply:
Yes, we should. I copied the same code over and forgot to delete those comments. Let me fix that